lib/less-browser/add-default-options.js

```diff
@@ -43,7 +43,5 @@ module.exports = function(window, options) {
         options.onReady = true;
     }
 
-    // TODO: deprecate and remove 'inlineJavaScript' thing - where it came from at all?
-    options.javascriptEnabled = (options.javascriptEnabled || options.inlineJavaScript) ? true : false;
-
+    options.javascriptEnabled = options.javascriptEnabled ? true : false;
```

Comment:
` ? true : false` isn't really necessary is it?  Or is this to force Boolean?  Do we have a style guide covering this case?  Maybe just `Boolean(options.javascriptEnabled)` would be more straightforward?

Reply:
I should just remove this line. I was just removing the `inlineJavaScript` thing, which was never used and shouldn't have gone in.